injective-common/README.md

```diff
@@ -1,129 +1,92 @@
-# Injective Foundational Modules
+# injective-common
 
-The Injective Foundational modules are Substreams modules extracting common data from the Injective blockchain.
+Common Mantra Substreams modules to extract events and transactions with indexing
+This package inherits from the generic Cosmos Foundational Modules.
 
-## Getting the substreams package
+## Usage
 
-### From Substreams.dev
+Usually, foundational modules are directly imported and used in other Substreams. All the official foundational modules are stored in [substreams.dev](`https://substreams.dev`).
 
-https://substreams.dev/streamingfast/injective-common/
+```yaml
+specVersion: v0.1.0
+package:
+  name: my_project
+  version: v0.1.0
 
+imports:
+  injective: https://spkg.io/streamingfast/injective-common-v0.2.4.spkg # Import the package from substreams.dev
```

Comment:
Here, I think we will need to change and have a cosmos-common instead of injective-common